lib/ansible/modules/remote_management/ucs/ucs_timezone.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: ucs_timezone
+
+short_description: Configures timezone on Cisco UCS Manager
+
+version_added: "2.7"
+
+description:
+    - Configures timezone on Cisco UCS Manager.
+    - Examples can be used with the UCS Platform Emulator U(https://communities.cisco.com/ucspe).
+
+options:
+    state:
+        description:
+            - If C(present), will set or update timezone.
+            - If C(absent), will unset timezone.
+        choices: [present, absent]
+        default: present
+
+    admin_state:
+        description: admin state indicates if the timezone confguration is enabled and being utilized by UCS Manager
+            or disabled and being ignored by UCS Manager
+        choices: ['disabled', 'enabled']
+        default: "enabled"
```

Comment:
Have you considered using YAML booleans for boolean values ?
I am not sure if @dsoper2 had been doing this already.
IMO it provides a more native look and feel.
We implemented this for ACI as well (where internally different values were used for booleans).
Beware though that some values may look boolean, but sometimes allow for other values (e.g. None, empty, or different flags, etc...).

Reply:
I have not seen where @dsoper2 has implemented this, I'm hesitant to change to YAML booleans as these are the actual values that admin_state can be set to.